components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m

```diff
@@ -80,7 +96,33 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  //  self.recordMode = YES;
+  self.recordMode = YES;
```

Comment:
This should be commented out

Reply:
Done, I was scrambling while getting off the train. Thanks!